modular_skyrat/modules/organs/code/stomach.dm

```diff
@@ -0,0 +1,7 @@
+/obj/item/organ/stomach/oversized
+	name = "huge guts"
+	desc = "Typically found in huge creatures, this monstrous engine has developed to be highly efficient, made to get an enormous amount of nutrients to an enormous eater."
+	icon = 'modular_skyrat/modules/organs/icons/stomach.dmi'
+	icon_state = "stomach_big"
+	maxHealth = 1.5 * STANDARD_ORGAN_THRESHOLD
+	metabolism_efficiency = 0.07
```

Comment:
What's the normal `metabolism_efficiency` at?

Reply:
0.05. This thing's basically comparable to a tier 2 cybernetic stomach.